packages/web-components/@sfx/autocomplete/test/unit/autocomplete.test.ts

```diff
@@ -36,19 +45,25 @@ describe('Autcomplete Component', () => {
       expect(superConnectedCallbackStub).to.have.been.called;
     });
 
-    it('should add an eventListener to the window', () => {
-      const windowAddEventListener = spy(window, 'addEventListener');
-
+    it('should add event listeners to the component and window', () => {
       autocomplete.connectedCallback();
 
       expect(windowAddEventListener).to.have.been.calledWith(
-        'sfx::autocomplete_received_results',
+        AUTOCOMPLETE_RECEIVED_RESULTS_EVENT,
         autocomplete.receivedResults
       );
     });
   });
 
   describe('disconnectedCallback', () => {
+    let windowRemoveEventListener;
+    let autocompleteRemoveEventListener;
+
+    beforeEach(() => {
+      windowRemoveEventListener = stub(window, 'removeEventListener');
+      autocompleteRemoveEventListener = stub(autocomplete, 'removeEventListener');
```

Comment:
This stub isn't being used.

Reply:
Same as above. The function under test is using it, so I'm stubbing it out here.

Reply:
Same as above. *sigh*